docs/projects/P2/1_teamprocess.md

```diff
@@ -106,33 +106,41 @@ During this meeting, discuss potential functional requirements of this project.
 
-In the body of each of these draft issues, provide a brief justification of its prioritization ranking that your team decided on. You should order all the user stories in this column from highest to lowest priority.
+Add your user stories into the **"User Stories"** column using the **“+ Add item”** button at the bottom, create draft issues for each of your user stories.In the body of each of these draft issues, provide a brief but concrete justification of its prioritization ranking that your team decided on. You should order all the user stories in this column from highest to lowest priority.
 
 #### Technical Requirements - Issues
 
-Now that you have your prioritized list, consider the *technical requirements* of the various user stories and collectively decide on which one(s) you will be focusing on over the next two sprints.
+Now that you have your prioritized list, consider the *technical requirements* of the various user stories and collectively decide on which one(s) you will be focusing on over the next two sprints. In this project (as like most projects), your team is aiming to maximize the amount of value you are delivering to your stakeholders given your team and resources provided.
+
+Your selected user stories should have relatively high priority based on your team's ranking, and you should actively take factors into consideration that may impact your development.
+
+!!! note "Selecting Appropriate User Stories"
+    Given the amount of variations in each team's user stories, it's hard to give concrete guideline on the number of user stories that a team needs to tackle. Teams could tackle 1 user story that requires major effort, or a few user stories that each requires lesser effort. 
+    
+    In general, we are expecting that user stories be selected given:
 
-Your selected user stories should have relatively high priority based on your team's ranking, and you should actively take factors into consideration that may impact your development, such as onboarding time to the repository and its code, technical expertise within your team, time commitments, etc.
+    - 2 sprints of 2 weeks each
+    - number of team members on your team
+    - assumption of 8 hours/week available per individual
 
-The feature(s) you plan to implement should affect **all three layers of the NodeBB codebase**:
+    The course staff **is happy to discuss this with your team during OH** and we highly recommend you do so if your team is unsure. We will also be providing you with feedback during your first sprint's [checkpoint presentation](http://127.0.0.1:8000/projects/P2/2_firstsprint/#checkpoint-presentation-50-pts).
```

Comment:
should replace 127.0.0.1 with relative URL or cmu-313.github.io

Reply:
thanks for the catch